docs/specs/TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Abstract
+
+Briefly describe the proposal
+
+# Motivation
+
+Why you think the proposal is necessary
+
+# Background
+
+Background information related to the proposal (current implementation, rationale for the implementation you will propose)
+
+# Proposal
+
+What you propose should be done
```

Comment:
Should we add something like this at the end 🤔?

```
#Possible negative drawbacks or complications

Optionally describe what are the complications that could go wrong while or after implementing this feature.
```

Reply:
I think that would go in the `background` section where you could contrast different implementation ideas